FAULT TYPE: OPENSTACK_THROW_EXCEPTION-INSTANCE
TARGET COMPONENT: /nova/compute/resource_tracker.py
TARGET CLASS: ResourceTracker
TARGET FUNCTION DEF: instance_claim+222;12
FAULT POINT: self.pci_tracker.claim_instance(context, pci_requests, instance_numa_topology)
